src/dotnet/UI.Blazor/Components/FeatureRequestModal.razor

```diff
@@ -34,9 +34,7 @@
         <textarea class="feedback-textarea border border-accent w-full" rows="5" @bind="_comment" @bind:event="oninput"></textarea>
     </div>
     <div class="mt-4">
-        <button type="button" @onclick="OnSubmit" class="signin-button">
-            <span>Send feedback</span>
-        </button>
+        <Button Class="btn btn-success" OnClick="OnSubmit" InnerText="Send feedback" ></Button>
```

Comment:
NOPE.
- any `<Button>` should have `btn` class by default
- so all you add is `btn-success` (i.e. extra classes)
- if you don't want `<Button>` to have `btn` class - just create a new control (not `<Button>`)!